_posts/2019-05-20-tiaas-feedback.md

```diff
@@ -0,0 +1,7 @@
+---
+site: freiburg
+tags: [training, tiaas]
+title: "Another successful TIaaS workshop"
```

Comment:
`Another successful Galaxy workshop in Melbourne` Maybe? Not sure the readers will know what TIaaS is ...